python/lsst/dax/apdb/apdbSchema.py

```diff
@@ -28,6 +28,7 @@
 
 from collections import namedtuple
 import logging
+import os.path
```

Comment:
`import os` is sufficient for `os.path` (though I saw some people prefer this style).